health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1009,99 +989,97 @@
   const topic = firstPersist
     ? config?.kafka?.KAFKA_SAVE_PROJECT_CAMPAIGN_DETAILS_TOPIC
     : config?.kafka?.KAFKA_UPDATE_PROJECT_CAMPAIGN_DETAILS_TOPIC;
-  delete request.body.CampaignDetails.codesTargetMapping;
+  delete requestBody.CampaignDetails.codesTargetMapping;
   const produceMessage: any = {
-    CampaignDetails: request?.body?.CampaignDetails,
+    CampaignDetails: requestBody.CampaignDetails,
   };
   await produceModifiedMessages(produceMessage, topic);
-  delete request.body.CampaignDetails.campaignDetails;
+  delete requestBody.CampaignDetails.campaignDetails;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)